src/com/facebook/buck/cli/PublishCommand.java

```diff
@@ -68,20 +78,25 @@
   private URL remoteRepo = null;
 
   @Option(
-      name = TO_MAVEN_CENTRAL_LONG_ARG,
-      usage = "Same as \"" + REMOTE_REPO_LONG_ARG + " " + Publisher.MAVEN_CENTRAL_URL + "\"")
-  private boolean toMavenCentral = false;
```

Comment:
this changes the logic and breaks existing use cases:

1) it requires `toMaven` when it otherwise would not. 
2) it makes new behavior default.
3) it changes flag names and logic.

Reply:
If backward compatibility is the preference, I can certainly make it happen by keeping all the existing flags and add another flag call `isMaven`. However, this new flag is mutually exclusive of the `to-maven-central`. It feels a bit redundant.

I can also set the default value of `toMaven` to be `true`, to keep the changes to existing use cases minimal.

Reply:
Breakage of existing users seems unnecessary here. `isMaven` does not need to be mutually exclusive with `to-maven-central` and `to-maven-central` can imply `isMaven=true` and fail if `isMaven=false`